cmd/entrypoint/watcher.go

```diff
@@ -28,11 +29,13 @@ func WatchAllTheThings(
 	fastpathCh chan<- *ambex.FastpathSnapshot,
 	clusterID string,
 	version string,
+	maxInterval time.Duration,
 ) error {
 	client, err := kates.NewClient(kates.ClientConfig{})
 	if err != nil {
 		return err
 	}
+	client.MaxAccumulatorInterval(maxInterval)
```

Comment:
Maybe a log line to help support team with debugging and allowing user to see it was set correctly?